test/parse_request.cc

```diff
@@ -2354,7 +2354,9 @@ TEST(ParseRequest, test_use_living_streets) {
   costing_with_defaults.emplace_back(Costing::pedestrian, kDefaultPedestrian_UseLivingStreets);
   costing_with_defaults.emplace_back(Costing::bicycle, kDefaultBicycle_UseLivingStreets);
 
-  for (const auto& [costing, default_value] : costing_with_defaults) {
```

Comment:
the changes here are to fix compiler warnings about c++17 that i was sick of seeing